lite/files/Instructions.md

```diff
@@ -0,0 +1,35 @@
+
+# Instructions for Using Oncoprint Data
+## Overview
+This notebook provides a visualization of mutation data using an oncoprint heatmap. This visualization helps in understanding the presence and distribution of genetic alterations across various samples.
```

Comment:
```suggestion
This notebook provides a visualization of mutation data using a heatmap. This visualization shows the presence of genetic alterations in each gene across the samples.
```

Reply:
bit subtle but the OncoPrint heatmap has a lot more visual glyphs etc, calling it a "vanilla" heatmap makes it a bit more clear users' shouldn't expect the full OncoPrint functionality